e107_languages/English/admin/lan_admin.php

```diff
@@ -425,6 +425,13 @@
 
 define("LAN_UI_USING_DATABASE_TABLE", "Using [x] database table");
 define("LAN_UI_TOTAL_RECORDS", "Total Records: [x]");
+define("LAN_UI_1_HOUR", "1 hour");
+define("LAN_UI_3_HOURS", "3 hours");
+define("LAN_UI_6_HOURS", "6 hours");
+define("LAN_UI_12_HOURS", "12 hours");
+define("LAN_UI_24_HOURS", "24 hours");
+define("LAN_UI_48_HOURS", "48 hours");
+define("LAN_UI_3_DAYS", "3 days");
```

Comment:
These terms are used only in admin/users.php and no where else in all of e107. Therefor they should be inside lan_users.php, not lan_admin.php